doc/source/train/user-guides/checkpoints.rst

```diff
@@ -258,197 +167,163 @@ appropriately in distributed training.
-checkpoints to disk), a :py:class:`~ray.train.CheckpointConfig` can be passed into
-``Trainer``.
+Ray Train provides some configuration options for checkpointing via :class:`~ray.train.CheckpointConfig`.
+The primary configuration is keeping only the top ``K`` checkpoints with respect to a metric.
+Lower-performing checkpoints are deleted to save storage space. By default, all checkpoints are kept.
 
 .. literalinclude:: ../doc_code/key_concepts.py
     :language: python
     :start-after: __checkpoint_config_start__
     :end-before: __checkpoint_config_end__
 
 
-.. seealso::
-
-    See the :class:`~ray.train.CheckpointConfig` API reference.
-
 .. note::
 
-    If you want to save the top-k checkpoints with respect to a metric via
+    If you want to save the top ``num_to_keep`` checkpoints with respect to a metric via
     :py:class:`~ray.train.CheckpointConfig`,
     please ensure that the metric is always reported together with the checkpoints.
 
 
+
+Using checkpoints after training
+--------------------------------
+
+The latest saved checkpoint can be accessed with :attr:`Result.checkpoint <ray.train.Result.checkpoint>`.
+
+The full list of persisted checkpoints can be accessed with :attr:`Result.best_checkpoints <ray.train.Result.best_checkpoints>`.
+If :class:`CheckpointConfig(num_to_keep) <ray.train.CheckpointConfig>` is set, this list will contain the best ``num_to_keep`` checkpoints.
+
+See :ref:`train-inspect-results` for a full guide on inspecting training results.
+
+:meth:`Checkpoint.as_directory <ray.train.Checkpoint.as_directory>`
+and :meth:`Checkpoint.to_directory <ray.train.Checkpoint.to_directory>`
+are the two main APIs to interact with Train checkpoints.
+
+
 .. _train-dl-loading-checkpoints:
 
-Loading checkpoints
--------------------
+Restore training state from a checkpoint
+----------------------------------------
+
+In order to enable fault tolerance, you should modify your training loop to restore
+training state from a :class:`~ray.train.Checkpoint`.
 
-:class:`Checkpoints <ray.train.Checkpoint>` can be accessed in the training function with :func:`ray.train.get_checkpoint <ray.train.get_checkpoint>`.
+The :class:`Checkpoint <ray.train.Checkpoint>` to restore from can be accessed in the
+training function with :func:`ray.train.get_checkpoint <ray.train.get_checkpoint>`.
 
-The checkpoint can be populated in two ways:
+The checkpoint returned by :func:`ray.train.get_checkpoint <ray.train.get_checkpoint>` is populated in two ways:
 
-1. It can be auto-populated, e.g. for :ref:`automatic failure recovery <train-fault-tolerance>` or :ref:`on manual restoration <train-restore-guide>`.
+1. It can be auto-populated as the latest reported checkpoint, e.g. for :ref:`automatic failure recovery <train-fault-tolerance>` or :ref:`on manual restoration <train-restore-guide>`.
 2. The checkpoint can be passed to the :class:`Trainer <ray.train.trainer.BaseTrainer>` as the ``resume_from_checkpoint`` argument.
```

Comment:
```suggestion
2. It can be manually populated by passing a checkpoint to the ``resume_from_checkpoint`` argument of a Ray :class:`Trainer <ray.train.trainer.BaseTrainer>`.
```